autoencoder.py

```diff
@@ -1,17 +1,32 @@
 """Implementation of feed forward autoencoder in TensorFlow 2.0"""
+
 from __future__ import absolute_import
 from __future__ import division
 from __future__ import print_function
 
 __version__ = '1.0.0'
 __author__ = 'Richard Ricardo'
 
+import numpy as np
 import tensorflow as tf
+import datetime
 from tensorflow.keras.layers import Dense
-from keras.models import Model
+
 
 class Autoencoder(tf.keras.Model):
-    def __init__(self, intermediate_dim, code_dim, original_dim, num_hidden_layers, encoder_decrements, decoder_increments):
+
+    def __init__(
+        self,
+        intermediate_dim,
+        code_dim,
+        original_dim,
+        num_hidden_layers,
+        encoder_decrements,
+        decoder_increments,
+        learning_rate=1e-3,
```

Comment:
Sorry, upon thinking about it, we can remove this, and maintain the optimizer outside the class.